engine/api/workflow/dao_node_run_job.go

```diff
@@ -13,10 +13,11 @@ import (
 	"github.com/ovh/cds/engine/api/cache"
 	"github.com/ovh/cds/engine/api/group"
 	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
 )
 
-// LoadNodeJobRunQueue load all workflow_node_run_job accessible
-func LoadNodeJobRunQueue(db gorp.SqlExecutor, store cache.Store, groupsID []int64, since *time.Time, statuses ...string) ([]sdk.WorkflowNodeJobRun, error) {
+// return query, groupID, statuses, isSharedInfraGroup
+func queryLoadNodeJobRunQueue(isCount bool, groupsID []int64, since *time.Time, statuses ...string) (string, string, *time.Time, string, bool) {
```

Comment:
i don't like it, i would prefer two different functions event if there is a bit of code duplication 
- or may be groupID stuff should be in a function
- or the query should be in a constant